MSAL/src/native_auth/network/parameters/MSALNativeAuthRequestContext.swift

```diff
@@ -59,4 +61,23 @@ final class MSALNativeAuthRequestContext: MSIDRequestContext {
             MSID_APP_VER_KEY: appVersion
         ]
     }
+
+    func setServerCorrelationId(_ serverCorrelationId: UUID?) {
+        guard let serverCorrelationId = serverCorrelationId else {
+            MSALLogger.log(level: .info, context: self, format: "correlationId not found in server response")
+            return
+        }
+
+        guard _correlationId != serverCorrelationId else {
+            return
+        }
+
+        let log = """
+                  Inconsistency between the correlationId sent by the SDK the one received in the response.
```

Comment:
Typo: Missing "and" between "SDK" and "the one" - should be "...the SDK **and** the one received..."